script/Ciel/Day 15/15_01B_CIEL15B_1.txt

```diff
@@ -2641,7 +2646,7 @@ She doesn't say anything.
 [sha:e5a63e9bf0d6b2a3b7b810beaf4a4591cd8d843c]{
 -- Page 156, Offset 42440.
 -- 声をだす必要はこれっぽっちも感じなくて、ただ、その体温を感じていた。
-I don't feel any need to speak either, and just bask in the warmth of her touch.
+Nor do I feel even the slightest need to speak, and instead just bask in the warmth of her touch.
```

Comment:
The nor construction makes it sound like his speaking need is different to her need. I'd just revert this one too.